privacyidea/api/periodictask.py

```diff
@@ -72,7 +72,7 @@ def list_nodes():
     """
     Return a list of available nodes
     """
-    nodes = get_privacyidea_nodes()
+    nodes = get_privacyidea_nodes(names_only=True)
```

Comment:
boolean function parameter is most of the time not a good idea because it indicates that the function has two behaviors, which, against most the time, should just be two separate functions.
Is there a reason not to make a function `get_privacyidea_node_names`?

Reply:
Done in 200d355

Reply:
Done in 658afd5